internal/controller/dpuoperatorconfig_controller_test.go

```diff
@@ -156,9 +156,13 @@ func startDPUControllerManager(ctx context.Context, client *rest.Config, wg *syn
 		LeaderElectionID: "1e46962d.openshift.io",
 	})
 	Expect(err).NotTo(HaveOccurred())
+	cwd, err := os.Getwd()
```

Comment:
I don't see how much we gain here when we have to have an OS call here to get the root directory.

Reply:
The main gain is that this PR removes the `os` as a dependency in the reconciler. That means that for the sake of readability, you will know that there is no use of env (i.e. no hidden input on that front) to the reconciler anywhere.

I agree that the test now depends on being run from the right directory. I will work on removing that.